app/css/Wavehand.css

```diff
@@ -2,20 +2,20 @@
 .button-rows {
   display: inline-block;
   width: 95%;
-  margin-left: 30px;
-  margin-top: 20px;
-  margin-bottom: 20px;
+  margin-left: calc(1.875rem + 1vw); /* Responsive to system settings and screen size */
+  margin-top: calc(1.25rem + 0.5vw); /* Responsive to system settings and screen size */
+  margin-bottom: calc(1.25rem + 0.5vw); /* Responsive to system settings and screen size */
 }
 
 .wave-hand-button {
-  padding: 10px 20px;
+  padding: calc(0.625rem + 0.25vw) calc(1.25rem + 0.5vw); /* Responsive padding */
   background-color: transparent;
   color: black;
   border: 1px solid #ccc;
-  border-radius: 20px;
+  border-radius: calc(1.25rem + 0.25vw); /* Responsive border-radius */
   cursor: pointer;
-  margin: 5px;
-  font-size: 20px;
+  margin: calc(0.3125rem + 0.25vw); /* Responsive margin */
+  font-size: calc(1.25rem + 0.25vw); /* Responsive font size, adjusts with system font settings and screen size */
```

Comment:
These numbers are *very* precise - how are they derived?

Reply:
I have some assumptions
    Root Font Size (html): 16px (which is common for most browsers).
    Viewport Width: 1200px (which is a typical desktop screen size).
 1. Margin: calc(0.3125rem + 0.25vw)

    0.3125rem:
        1rem = 16px (root font size).
        0.3125rem = 16px * 0.3125 = 5px.

    0.25vw:
        1vw = 1% of the viewport width.
        Viewport width = 1200px.
        0.25vw = 1200px * 0.25 / 100 = 3px.

    Total Margin:
        calc(0.3125rem + 0.25vw) = 5px + 3px = 8px.

So, in this example, the margin would be 8px.
2. Font Size: calc(1.25rem + 0.25vw)

    1.25rem:
        1rem = 16px (root font size).
        1.25rem = 16px * 1.25 = 20px.

    0.25vw:
        1vw = 1% of the viewport width.
        Viewport width = 1200px.
        0.25vw = 1200px * 0.25 / 100 = 3px.

    Total Font Size:
        calc(1.25rem + 0.25vw) = 20px + 3px = 23px.

Reply:
Resolved